lib/css/CssParser.js

```diff
@@ -815,12 +859,29 @@ class CssParser extends Parser {
 						}
 						break;
 					}
+					case CSS_MODE_IN_ICSS_IMPORT: {
+						scope = CSS_MODE_TOP_LEVEL;
+						const dep = new ConstDependency("", [currIcssImport.start, end]);
+						module.addPresentationalDependency(dep);
+						icssImportsData.push(currIcssImport);
+						currIcssImport = undefined;
+					}
 				}
 				return end;
 			},
 			identifier: (input, start, end) => {
 				switch (scope) {
 					case CSS_MODE_IN_BLOCK: {
+						if (icssImportsData) {
+							const dataIndex = icssImportsData.findIndex(data =>
+								data.importMap.get(input.slice(start, end))
+							);
```

Comment:
Never use `findIndex`, if you want to use it means you do something wrong, wrong place when you replace (too later)